frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -1,20 +1,49 @@
 import { testName } from '@console/internal-integration-tests/protractor.conf';
-import { CloudInitConfig, BaseVMConfig, StorageResource } from './types';
+import { ConfigMapKind, SecretKind, ServiceAccountKind } from '@console/internal/module/k8s';
+import { CloudInitConfig, Disk } from '../types/types';
 import {
   STORAGE_CLASS,
   commonTemplateVersion,
-  NIC_MODEL,
-  NIC_TYPE,
-  DISK_INTERFACE,
   KUBEVIRT_STORAGE_CLASS_DEFAULTS,
   KUBEVIRT_PROJECT_NAME,
   COMMON_TEMPLATES_NAMESPACE,
   COMMON_TEMPLATES_REVISION,
+} from '../utils/constants/common';
+import {
+  getRandomMacAddress,
+  getResourceObject,
+  resolveStorageDataAttribute,
+} from '../utils/utils';
+import { Flavor, ProvisionSource } from '../utils/constants/wizard';
+import {
+  NIC_MODEL,
+  NIC_TYPE,
+  DISK_INTERFACE,
   DISK_SOURCE,
-} from './consts';
-import { getRandomMacAddress, getResourceObject, resolveStorageDataAttribute } from './utils';
-import { Flavor, OperatingSystem, WorkloadProfile } from './constants/wizard';
-import { ConfigMapKind, SecretKind, ServiceAccountKind } from '@console/internal/module/k8s/types';
+  DISK_DRIVE,
+} from '../utils/constants/vm';
+
+export const provisionSources = {
+  [ProvisionSource.URL]: {
+    method: ProvisionSource.URL,
+    source:
+      'http://cnv-qe-server.rhevdev.lab.eng.rdu2.redhat.com/files/cnv-tests/cirros-images/cirros-0.4.0-x86_64-disk.raw',
```

Comment:
I think I saw some issues with the public one, because our clusters download this image from the same IP address, and sometimes it seems that download.cirros-cloud.net rate limits the requests. I'll change it to public one, if there are issues, we can always use the internal mirror.